apps/patient/src/views/Pharmacy.tsx

```diff
@@ -267,7 +265,10 @@ export const Pharmacy = () => {
       return;
     }
 
-    const preparedPharmacies: PharmacyWithHours[] = pharmaciesResult.map(preparePharmacy);
+    const preparedPharmacies: PharmacyWithHours[] = preparePharmacyOptions(
+      pharmaciesResult,
+      topRankedPharmacies
+    );
```

Comment:
is it the top ranked pharmacies would be duplicated?

Reply:
yes